woocommerce/woocommerce
# Introduce the new front-end filter blocks in beta state

### Changes proposed in this Pull Request:

In this PR all the feature gating around the new front-end filter blocks is removed, allowing users to use the new front-end filter blocks. This combined with https://github.com/woocommerce/woocommerce/pull/44487 will release these new blocks to production in a beta state.



### How to test the changes in this Pull Request:

1. Do a production build of WooCommerce blocks (or run this on JN - which is prod by default right now)
2. Search for the new filter blocks in the GB inserter such as `Product Filters: ...` and see that these filter blocks are available for insertion.
3. Smoke test that inserting and using them works as expected.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Release the new product collection filter blocks in a beta state.


#### Comment 

</details>
